test/app/controllers/albums.post.test.js

```diff
@@ -0,0 +1,100 @@
+const request = require('supertest');
+const { factory } = require('factory-girl');
+const bcrypt = require('bcrypt');
+const nock = require('nock');
```

Comment:
I think that you wouldn't use another package for mock request (nock) because  jest already contains one ( check [this](https://jestjs.io/docs/en/mock-functions.html))

Reply:
also, if you can mock request-promise... you can mock bcrypt too :smile: